src/jl_uv.c

```diff
@@ -27,8 +27,23 @@
 #include <io.h>
 #define write _write
 #endif
-#if defined(_COMPILER_MICROSOFT_) && !defined(_Static_assert)
-#define _Static_assert static_assert
+
+#ifndef static_assert
+#  ifdef __cplusplus
+#    if __cplusplus < 201103L
+#      define static_assert
+#    endif
+#  else
+#    define static_assert
```

Comment:
I think you are right I was thinking about defining static alert as alert but removed it